### PR TITLE
minor corrections and two changes in the examples

### PR DESCRIPTION
**Pull request by kzqureshi**
There were some typos in the documentations and few changes are done.

### CODE REVIEW

**Comment by lang-m** [comment]
@samjrholt Could you have a look at this and help @kzqureshi with any changes that might be required? Looks like these changes are on a slightly older version of the notebooks, hence the merge conflicts. It might be easier to manually reapply it onto the current master.

**Review by samjrholt — APPROVED** [review]
Just one minor suggestion but then please merge it.

File: examples/choosing-runner.ipynb [hunk 1]
```diff
@@ -12,7 +12,7 @@
...
+    "In this tutorial, we show how to choose different \"runners\" to run your simulations. This is helpful if you want to change OOMMF installation you want to use. This is in particular helpful if you want to run OOMMF inside Docker, which allows us to run simulations on a \"small linux machine\", which is automatically pulled from the cloud, simulations run inside, and in the end it is destroyed automatically. This all happens in the background and requires no special assistance from the user. In order to use Docker, we need to have it installed on our machine - you can download it here: https://www.docker.com/products/docker-desktop.\n",
```
Comment (lang-m) [inline examples/choosing-runner.ipynb:15]: I think I would keep the old version for this one.

Comment (lang-m) [reply examples/choosing-runner.ipynb:15]: @kzqureshi Please have a look. Once this is resolved we can merge.

Comment (kzqureshi) [reply examples/choosing-runner.ipynb:15]: Resolved from my side.

**Pull request merged**